src/sdk/uservarmanager.cpp

```diff
@@ -174,11 +117,11 @@ wxString UserVariableManager::Replace(const wxString& variable)
             msg.Printf(_("In the currently active set, Code::Blocks does not know\n"
                          "the global compiler variable \"%s\".\n\n"
                          "Please define it."), package.wx_str());
-            InfoWindow::Display(_("Global Compiler Variables"), msg , 8000, 1000);
-            UsrGlblMgrEditDialog d;
-            d.AddVar(package);
-            PlaceWindow(&d);
-            d.ShowModal();
+
+            m_ui->DisplayInfoWindow(_("Global Compiler Variables"), msg);
+            wxArrayString ar;
```

Comment:
Prefer std::vector in new code. I think.

Reply:
i replaced the wxArrayString with a std::set<wxString> in d88ee6c96